client/src/actions/index.ts

```diff
@@ -36,29 +40,27 @@ async function userColorsFetchAndLoad(dispatch: any) {
   );
 }
 
-async function schemaFetch() {
-  return fetchJson("schema");
+async function schemaFetch(): Promise<{ schema: Schema }> {
+  return fetchJson<{ schema: Schema }>("schema");
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-async function configFetch(dispatch: any) {
-  return fetchJson("config").then((response) => {
-    const config = { ...globals.configDefaults, ...response.config };
+async function configFetch(dispatch: Dispatch) {
+  const response = await fetchJson<{ config: globals.Config }>("config");
+  const config = { ...globals.configDefaults, ...response.config };
 
-    setGlobalConfig(config);
+  setGlobalConfig(config);
 
-    dispatch({
-      type: "configuration load complete",
-      config,
-    });
-    return config;
+  dispatch({
+    type: "configuration load complete",
+    config,
   });
+  return config;
 }
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
 async function userInfoFetch(dispatch: any) {
   return fetchJson("userinfo").then((response) => {
-    const { userinfo: userInfo } = response || {};
+    const { userinfo: userInfo } = (response || {}) as { userinfo: unknown };
```

Comment:
while this shuts up tsc/eslint, it doesn't really do much for typing (it is no better than suppressing the lint error).

why not define the full interface for user info, and declare that as a return type?

Reply:
Fixed!